www/breakdownTimeline.php

```diff
@@ -3,305 +3,28 @@
-                    echo "eventsIdle.setValue($index, 0, '$type');\n";
-                    echo "eventsIdle.setValue($index, 1, $time);\n";
-                    $group = 'Other';
-                    if (array_key_exists($type, $mapping)) {
-                        $group = $mapping[$type];
-                    }
-                    $color = $groupColors[$group];
-                    echo "eventColors.push('$color');\n";
-                    $index++;
-                }
-            }
-            ?>
-            var viewGroupsIdle = new google.visualization.DataView(groupsIdle);
-            viewGroupsIdle.setColumns([0, 1]);
-
-            var tableGroupsIdle = new google.visualization.Table(document.getElementById('tableGroupsIdle'));
-            tableGroupsIdle.draw(viewGroupsIdle, {showRowNumber: false, sortColumn: 1, sortAscending: false});
-
-            var pieGroupsIdle = new google.visualization.PieChart(document.getElementById('pieGroupsIdle'));
-            pieGroupsIdle.draw(viewGroupsIdle, {width: 450, height: 300, title: 'Processing Categories', colors: groupColors});
-
-
-            var viewEventsIdle = new google.visualization.DataView(eventsIdle);
-            viewEventsIdle.setColumns([0, 1]);
-
-            var tableEventsIdle = new google.visualization.Table(document.getElementById('tableEventsIdle'));
-            tableEventsIdle.draw(viewEventsIdle, {showRowNumber: false, sortColumn: 1, sortAscending: false});
-
-            var pieEventsIdle = new google.visualization.PieChart(document.getElementById('pieEventsIdle'));
-            pieEventsIdle.draw(viewEventsIdle, {width: 450, height: 300, title: 'Processing Events', colors: eventColors});
-        }
-        </script>
-    </body>
-</html>
-
-<?php
-function rgb2html($r, $g = -1, $b = -1)
-{
-    if (is_array($r) && sizeof($r) == 3) {
-        list($r, $g, $b) = $r;
-    }
-
-    $r = intval($r);
-    $g = intval($g);
-    $b = intval($b);
-
-    $r = dechex($r < 0 ? 0 : ($r > 255 ? 255 : $r));
-    $g = dechex($g < 0 ? 0 : ($g > 255 ? 255 : $g));
-    $b = dechex($b < 0 ? 0 : ($b > 255 ? 255 : $b));
-
-    $color = (strlen($r) < 2 ? '0' : '') . $r;
-    $color .= (strlen($g) < 2 ? '0' : '') . $g;
-    $color .= (strlen($b) < 2 ? '0' : '') . $b;
-    return '#' . $color;
-}
-?>
+include 'common.inc';
+require_once('breakdown.inc');      // No affect removing
+require_once('contentColors.inc');  // No affect removing
+require_once('waterfall.inc');      // No affect removing
+require_once('page_data.inc');      // No affect removing
+
+//region HEADER 
+$tab = 'Test Result';
+$subtab = 'Processing';
+include_once 'header.inc';
+//endregion
+
+//region template
+require_once __DIR__ . '/resources/view.php';
+echo view('pages.breakdownTimeline', [
+    //'test_results_view' => true, // Not Sure if we need this
+    'body_class' => 'result',
+    // 'results_header' => $results_header, // Null Value, Not sure if we need this
```

Comment:
![image](https://user-images.githubusercontent.com/33207323/197269791-b8923d10-90ea-4963-ba3a-484c91189e94.png)

I see what you mean, it was to fix the layout of the header. Currently without using $result_header my header looks like the image above.

Would you still recommend I pass in $result_header like you did in consolelog.php even though it is ok?
like this ?
```
ob_start();
define('NOBANNER', true); // otherwise Twitch banner shows 2x
$tab = 'Test Result';
$subtab = 'Processing';
include_once 'header.inc';
$results_header = ob_get_contents();
ob_end_clean();
```